### PR TITLE
Removed the checks. [skip appveyor]

### PR DESCRIPTION
**Pull request by pelson**
xref: https://github.com/conda-forge/erlang-feedstock/issues/1

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ export ERL_TOP="$(pwd)"
 ./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}" --without-javac \
   --with-libatomic_ops="${PREFIX}" --enable-m${ARCH}-build
 make
-make release_tests
+#make release_tests
 cd "${ERL_TOP}/release/tests/test_server"
 ${ERL_TOP}/bin/erl -s ts install -s ts smoke_test batch -s init stop
```
Comment (jakirkham) [inline recipe/build.sh]: So these two lines above are tests too. Even though they don't look it.

Comment (pelson) [inline recipe/build.sh]: They did look it, but I'm going to narrow it down slowly.

**Comment by jakirkham** [comment]
So, CircleCI failed to pull the Docker image. That is probably a fluke. However, Travis CI failed to run the other two test lines because the directory doesn't exist. Maybe we need to comment all three lines after all?

File: recipe/build.sh (hunk 1 above)
Comment (jakirkham) [inline recipe/build.sh]: Yeah, so those 2 lines will fail without running `make release_tests` though.

Comment (jakirkham) [inline recipe/build.sh]: Commented them too. Let's see what happens now.

**Comment by jakirkham** [comment]
Also this appears to be an interesting note in the [docs]( http://erlang.org/doc/installation_guide/INSTALL.html#id61759 ) (under testing).

> On builds without crypto, ssl and ssh there is a failed test case for undefined functions. Verify that the failed test case log only shows calls to skipped applications.

**Comment by jakirkham** [comment]
Yep that fixes it. Will put this in a formal PR.

**Comment by jakirkham** [comment]
Removed the Linux build from this (build number 3) as it doesn't correspond to a proper state of the recipe. There is already a Linux build from `master` with build number 5.

The OS X build already existed so was unaffected by these changes.

**Pull request closed without merging**